mmcv/runner/fp16_utils.py

```diff
@@ -63,7 +63,7 @@ def cast_tensor_type(inputs, src_type, dst_type):
         return inputs
 
 
-def auto_fp16(apply_to=None, out_fp32=False):
+def auto_fp16(apply_to=None, out_fp32=False, supported_types=[]):
```

Comment:
Add `supported_types` to line 79 and append `New in version 1.4.8.` in the end.

https://github.com/open-mmlab/mmcv/blob/6289b6f922f3b251791ef7cb5fd05d0c900b4750/mmcv/runner/hooks/logger/text.py#L52